### PR TITLE
Added template branch requirement.

### PR DESCRIPTION
**Pull request by Blackbaud-BobbyEarl**
#950

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-template-branch-requirement](https://developer.blackbaud.com/skyux?_sp.spa=pr-template-branch-requirement)

File: src/stache/includes/cli/template.html [hunk 1]
```diff
@@ -17,3 +17,7 @@
     <sky-code>-t &lt;ANY-GIT-URL&gt;</sky-code> — Allows for complete flexibility by cloning any valid Git URL.
   </li>
 </ul>
+
+<sky-alert>
+  Please note a versioned branch name using the <sky-code>[MAJOR_VERSION].x.x</sky-code> format is required to exist in the template git repository. At the time of writing, the <sky-code>4.x.x</sky-code> branch must exist.
```
Comment (blackbaud-johnly) [inline src/stache/includes/cli/template.html:22]: - I'd delete `Please note`. Our style is to generally avoid 'please', and we formatted this as an alert, so I think that implicitly tells users to `note` the content.
- I'd change `using` to either `with` or `that uses`.
- I'd delete `to exist`. I think the meaning here is same without it.
- Should `template` be `template's`? It seems like.
- I'd change `repository` be `repo`. (I know they are the same thing, but we generally go with `repo`.)
- What do you think of changing `At the time of writing, the...` to `For example, as of SKY UX version 4.0, the...`?

File: src/stache/includes/cli/template.html [hunk 2]
```diff
@@ -19,5 +19,5 @@
 </ul>
 
 <sky-alert>
-  A versioned branch name that uses the <sky-code>[MAJOR_VERSION].x.x</sky-code> format is required in the template's repo. At of SKY UX 4, the <sky-code>4.x.x</sky-code> branch must exist.
+  A versioned branch name that uses the <sky-code>[MAJOR_VERSION].x.x</sky-code> format is required in the template's repo. As of SKY UX 4, the <sky-code>4.x.x</sky-code> branch must exist.
```
Comment (Blackbaud-BobbyEarl) [inline src/stache/includes/cli/template.html:22]: HAH, sorry.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-template-branch-requirement](https://developer.blackbaud.com/skyux?_sp.spa=pr-template-branch-requirement)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-template-branch-requirement](https://developer.blackbaud.com/skyux?_sp.spa=pr-template-branch-requirement)

**Pull request merged**